examples/example2/script.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Initialize Mollie Components instance
+ */
+const mollie = Mollie("pfl_saaa222aaa3", {
```

Comment:
Would be nice to link to https://www.mollie.com/dashboard/developers/api-keys so they know where to find the profile ID.

Reply:
👍 Added a comment